lib/grammers-client/src/types/message.rs

```diff
@@ -388,6 +388,50 @@ impl Message {
         self.raw.reply_markup.clone()
     }
 
+    /// Collect all inline callback buttons. Those buttons often appeare in bot chat.
+    /// Buttons could be clicked by [`click_callback_button`]
+    pub fn callback_buttons(&self) -> Option<Vec<tl::types::KeyboardButtonCallback>> {
```

Comment:
Only handle KeyboardButtonCallback, other KeyboardButton types cannot handle.

Reply:
I was thinking more to return a custom type of buttons, so that `.click()` could be called directly on the one you need.

By the way, might be good to call this "inline buttons". The term in your code is used by the API, but I believe the other term is more common in the applications? Not sure what the bot API calls it.